std_msgs/Header header

float64 system_time
uint8   number_of_svs
uint8   processing_mode
uint16  gps_week_number
float64 gps_time_week
float64[3] position_rms
float64[3] velocity_rms
float64[3] attitude_rms
